### ISSUE TITLE
Options button is not reflective on the purpose of the button

### ISSUE THREAD

**Issue by TN032688** [cerner/terra-core#3527]
Accessibility 

The button that opens the Appointment Details modal is labeled Options.
Audit Comments: "Options" button is not reflective of the purpose of the button.

Remediation Proposal:
The "Options" button should change to "Appointment Detail" to match the title of the resulting modal. The Dropdown menu caret should be identified as "Appointment options" (or something similar). Both buttons should be programmatically associated to the actual appointment for user context.

## Steps to Reproduce

1.
2.
3.
4.

## Environment

* Component Name and Version:
* Browser Name and Version:
* Node/npm Version: [e.g. Node 8/npm 5]
* Webpack Version:
* Operating System and version (desktop or mobile):

**Comment by benbcai** [comment]
The `primaryOptionLabel` prop should be used to set the button label. I'm closing this issue for now.

**Comment by neilpfeiffer** [comment]
Closed to work with the consumer to resolve their implementation issue.

**Comment by scottwilmarth** [comment]
@neilpfeiffer and @benbcai this is actually an issue that should be addressed. There are multiple aspects to the split button that need to be resolved. First, the buttons are related yet have no programmatic connection. Second, if this split button was used multiple times on the page as "reply" and "more options" each of the buttons would need to be programmatically associated with content that provides more context about "what" are they replying to. The consumer team is using this button as "options" and "more options" for each appointment a consumer user may haveThe screen reader user would have no idea about what appointment the options or more options button is related to. It does not appear to me that the consuming teams have a way to make the programmatic connection between the content and the buttons. They must either visually hide text within the labels or provide a programmatic connection. Can either of you confirm or deny if either ability is provided. @ChaseSinclair
